### PR TITLE
[WIP] Adjusting doco to reflect changes made in Rabbit Transport

### LINKED ISSUE #1356: Updating configuration doco to correspond to RabbitMQ changes
Connects to https://github.com/Particular/docs.particular.net/issues/1378

This is a high level issue to track changes that need to be done to documentation related to RabbitMQ changes that were made to support NSB v6.

Upgrade guide for RabbitMQ will be tracked in a separate issue.

cc: @bording

### PR DESCRIPTION
**Pull request by WojcikMike**
Resolve #1356 

This is work in progress to track changes to the doco. I will work closely with @bording to properly address all the changes.

Connects to #1356

### CODE REVIEW

File: nservicebus/rabbitmq/configuration-api.md [hunk 1]
```diff
@@ -28,11 +28,22 @@ Below is the full list of connection string options. Note that they are separate
  * `Password`: The password when connecting. Defaults to `guest`
  * `RequestedHeartbeat`: The interval for the heartbeats between the client and the server. Defaults to `5` seconds
  * `DequeueTimeout` The time period allowed for the dequeue strategy to dequeue a message. Defaults to `1` second
+
+NOTE: In Version 6 and above `DequeueTimeout` is deprecated.
```
Comment (SimonCropp) [inline nservicebus/rabbitmq/configuration-api.md]: you mean version 4

Comment (WojcikMike) [inline nservicebus/rabbitmq/configuration-api.md]: Correct, I fixed it.

**Comment by indualagarsamy** [comment]
@bording - can you review this?

**Comment by bording** [comment]
This work only began to be started before the WIP limit, so none of the Rabbit docs are ready yet.

**Comment by SimonCropp** [comment]
@WojcikMike  @bording can u get the current state of this to a mergable state? Or should i close it?

**Comment by WojcikMike** [comment]
@SimonCropp the pr is in good enough state to be merged. We can continue work later.

**Comment by SimonCropp** [comment]
@WojcikMike can u fix the build error https://builds.particular.net/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=158512#_focus=2560

**Comment by WojcikMike** [comment]
@SimonCropp  Should be fine now, I messed rebase and merge.

**Comment by SimonCropp** [comment]
merged manually here https://github.com/Particular/docs.particular.net/commit/a0c5fdcee0c774f447f6550e855f7447a685718b

**Pull request closed without merging**